src/vma/proto/neighbour.cpp

```diff
@@ -1150,13 +1153,15 @@ bool neigh_entry::priv_get_neigh_state(int & state)
 bool neigh_entry::priv_get_neigh_l2(address_t & l2_addr)
 {
 	netlink_neigh_info info;
+	char str_addr[INET_ADDRSTRLEN];
 
 	if (m_is_loopback) {
 		memcpy(l2_addr, m_p_dev->get_l2_address()->get_address(), m_p_dev->get_l2_address()->get_addrlen());
 		return true;
 	}
 
-	if (g_p_netlink_handler->get_neigh(inet_ntoa(m_dst_addr.sin_addr), m_p_dev->get_if_idx(), &info)){
+	if (inet_ntop(AF_INET, &(m_dst_addr.sin_addr), str_addr, sizeof(str_addr)) &&
```

Comment:
Result shoud be the same.
`if (NULL != inet_ntop(...))`  or `if (inet_ntop(...))`